sdk/eventhub/Azure.Messaging.EventHubs/stress/stress-test-resources.bicep

```diff
@@ -0,0 +1,95 @@
+@description('The url suffix to use when creating storage connection strings.')
+param storageEndpointSuffix string = environment().suffixes.storage
+
+@description('The maximum duration, in minutes, that a single test is permitted to run before it is considered at-risk for being hung.')
+param perTestExecutionLimitMinutes string = '15'
+
+@description('The client OID to grant access to test resources.')
+param testApplicationOid string
+
+@description('The location of the resource group.')
+param location string = resourceGroup().location
+
+var eventHubsNamespace_var = 'eh-${resourceGroup().name}'
+var defaultSASKeyName = 'RootManageSharedAccessKey'
+var eventHubsAuthRuleResourceId = resourceId('Microsoft.EventHub/namespaces/authorizationRules', eventHubsNamespace_var, defaultSASKeyName)
+
+var ehBufferedProducer = 'bufferedproducertest'
+var bufferproducerPartitions = 10
```

Comment:
Do you have a preference about how you'd like to configure this? We used to support env-specific `parameters.json` files for ARM template deployment, but it wasn't used to I removed it.

Reply:
No preference beyond the standard ARM template experience.   If we can make them end up being required parameters that are only injected via params file when we're running on the platform, that would allow interactive prompting when deploying locally or via the portal.

Reply:
Do you think it's likely that we would also need different config values per "scenario" ?

Reply:
It'd be good to have that flexibility; depending on the scenario, having individual control over things like partition count or number of consumer groups would potentially be needed.

Reply:
Makes sense. I've filed https://github.com/Azure/azure-sdk-tools/issues/3208 for this.